iothub/device/samples/PnpDeviceSamples/TemperatureController/CustomJsonSerializer.cs

```diff
@@ -27,11 +27,6 @@ public override T DeserializeToType<T>(string stringToDeserialize)
             return JsonSerializer.Deserialize<T>(stringToDeserialize, s_options);
         }
 
-        public override bool CheckWritablePropertyResponseType(object typeToCheck)
```

Comment:
I think I'm okay with removing the base class and this check method. But we will need to make sure that our documentation is crystal clear around the WritablePropertyResponse.